src/open_producten/producttypen/admin/prijs.py

```diff
@@ -1,40 +1,63 @@
 from datetime import date
 
+from django import forms
 from django.contrib import admin
-from django.core.exceptions import ValidationError
-from django.forms import BaseInlineFormSet
-from django.utils.translation import gettext_lazy as _
+
+from open_producten.producttypen.models.validators import validate_prijs_optie_xor_regel
 
 from ..models import Prijs, PrijsOptie
+from ..models.prijs import PrijsRegel
+
+
+class PrijsOptieInline(admin.TabularInline):
+    model = PrijsOptie
+    extra = 1
+    fields = ("bedrag", "beschrijving")
 
 
-class PrijsOptieInlineFormSet(BaseInlineFormSet):
+class PrijsRegelInline(admin.TabularInline):
+    model = PrijsRegel
+    autocomplete_fields = ("dmn_config",)
+    extra = 1
+    fields = ("dmn_config", "dmn_tabel_id", "beschrijving")
+
+
+class PrijsAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = Prijs
+        fields = "__all__"
+
+    def get_entry_count(self, inline: str, unique_field: str) -> int:
+        # assert self.data.get(f"{inline}-0-{unique_field}") is not None
```

Comment:
Trailing comment, assert statements are ignored when running the app in uwsgi, so if this still applies you could uncomment it